src/components/program/ProgramsNavbar.tsx

```diff
@@ -2,32 +2,78 @@ import React from 'react';
 import { useRouteMatch, NavLink } from 'react-router-dom';
 import Style from './ProgramsNavbar.module.css';
 
-const programStages: {
-  name: string;
-  key: string;
-}[] = [
-  { name: 'Open', key: 'open' },
-  { name: 'Matching', key: 'matching' },
-  { name: 'Scheduled', key: 'scheduled' },
-  { name: 'In Progress', key: 'inprogress' },
-  { name: 'Closed', key: 'closed' },
-  { name: 'Canceled', key: 'canceled' },
-];
+// const programStages: {
+//   name: string;
+//   key: string;
+//   styleparameter:string;
+// }[] = [
+//   { name: 'Open', key: 'open', styleparameter: 'start' },
+//   { name: 'Matching', key: 'matching', styleparameter: 'mid' },
+//   { name: 'Scheduled', key: 'scheduled', styleparameter: 'mid' },
+//   { name: 'In Progress', key: 'inprogress', styleparameter: 'mid' },
+//   { name: 'Closed', key: 'closed', styleparameter: 'mid' },
+//   { name: 'Canceled', key: 'canceled', styleparameter: 'end' },
+// ];
 
 function ProgramsNavbar() {
   const { url } = useRouteMatch();
   return (
     <div className={Style.programsNavbar}>
-      {programStages.map((programStage) => (
+      <NavLink
+        key="open"
+        activeClassName={Style.start}
+        to={`${url}/open`}
+        data-testid="open"
+      >
+        Open
+      </NavLink>
+
+      <NavLink
+        key="matching"
+        activeClassName={Style.mid}
+        to={`${url}/matching`}
+        data-testid="matching"
+      >
+        Matching
+      </NavLink>
+
+      <NavLink
+        key="scheduled"
+        activeClassName={Style.mid}
+        to={`${url}/scheduled`}
+        data-testid="scheduled"
+      >
+        Scheduled
+      </NavLink>
+
+      <NavLink
+        key="inprogress"
+        activeClassName={Style.mid}
+        to={`${url}/inprogress`}
+        data-testid="inprogress"
+      >
+        In Progress
+      </NavLink>
+
+      <NavLink
+        key="closed"
+        activeClassName={Style.end}
+        to={`${url}/closed`}
+        data-testid="closed"
+      >
+        Closed
+      </NavLink>
+
+      {/* {programStages.map((programStage) => (
         <NavLink
-          activeClassName={Style.active}
-          to={`${url}/${programStage.key}`}
           key={programStage.key}
+          activeClassName={Style.activess}
+          to={`${url}/${programStage.key}`}
           data-testid={programStage.key}
         >
           {programStage.name}
         </NavLink>
-      ))}
+      ))} */}
```

Comment:
You can still map like Pratik did, but you'd probably have to add an optional property activeClassName. If activeClassName is specified use the specified one, otherwise use the default `Style.mid`